src/main/scala/io/druid/indexer/spark/SparkBatchIndexTask.scala

```diff
@@ -407,6 +414,46 @@ object SparkBatchIndexTask
         }
       )
 
+      log.info("Initializing emitter for metrics")
+
+      val service = task.getProperties.getProperty("com.metamx.emitter.service", "druid/batch/index")
+      val host = task.getProperties.getProperty("com.metamx.emitter.host", "127.0.0.1")
+
+      val emitter = new ServiceEmitter(service, host, createEmitter(task.getProperties, lifecycle))
```

Comment:
Oh, another thing is that I would like to configure service name but seems like right now it will have a hard-coded service name of `druid-indexer`. Will try modifying the injector and other parts to have a configurable service name for the `ServiceEmitter` and other stuff.